test/kennel/syncer_test.rb

```diff
@@ -609,7 +614,7 @@ def expect_gets(answer)
 
       it "refuses to update tracking on resources with ids since they would be deleted by other updates" do
         expected << monitor("a", "b", foo: "bar", id: 123)
-        monitors << monitor_api_response("a", "b", id: 123).merge(message: "An innocent monitor -- Managed by kennel b:b")
+        monitors << monitor_api_response("a", "b", id: 123).merge(message: "An innocent monitor -- Managed by kennel b:b", tracking_id: "b:b")
```

Comment:
Isn't this the same as: ?

```suggestion
        monitors << monitor_api_response("b", "b", id: 123)
```

Reply:
yeah, good catch ... idk how I ended in that rabbithole